eos/utils/kinematic.cc

```diff
@@ -54,7 +64,7 @@ namespace eos
                 _imp->variables_map[v->first] = index;
```

Comment:
Can you turn the C++03 loop into a range-based for loop, please?

Reply:
In line 57? Sure can do!

Reply:
Done.